src/hu/elte/txtuml/stdlib/io/File.java

```diff
@@ -0,0 +1,1019 @@
+package hu.elte.txtuml.stdlib.io;
+
+import java.io.FileFilter;
+import java.io.FilenameFilter;
+import java.io.IOException;
+
+import hu.elte.txtuml.api.model.Collection;
+import hu.elte.txtuml.api.model.external.ExternalClass;
+
+public interface File extends ExternalClass{
+
+	/**
+     * This abstract pathname's normalized pathname string. A normalized
+     * pathname string uses the default name-separator character and does not
+     * contain any duplicate or redundant separators.
+     *
+     * @serial
+     */
+	public final String pathSeparator = java.io.File.pathSeparator;
```

Comment:
These fields (`pathSeparator`, `pathSeparatorChar`, `separator`, `separatorChar`) should be `static`.